packages/zowe-explorer/src/job/ZosJobsProvider.ts

```diff
@@ -200,15 +200,7 @@ export class ZosJobsProvider extends ZoweTreeProvider implements IZoweTree<IZowe
     public async delete(node: IZoweJobTreeNode) {
         try {
             await ZoweExplorerApiRegister.getJesApi(node.getProfile()).deleteJob(node.job.jobname, node.job.jobid);
-            vscode.window.showInformationMessage(
-                localize("deleteJob.job", "Job ") +
-                    node.job.jobname +
-                    "(" +
-                    node.job.jobid +
-                    ")" +
-                    localize("deleteJob.delete", " deleted")
-            );
-            this.removeFavorite(this.createJobsFavorite(node));
+            await this.removeFavorite(this.createJobsFavorite(node));
```

Comment:
@crawr I am wondering if the children error is popping up due to it not being in favorites as I reran again and saving 2 of my 3 jobs into favorites then deleted all 3 and error only popped up once.  Before I just deleted 3 jobs and non were in favorites and the pop up occurred 3 times.

Reply:
this does call children if you change this:
![image](https://user-images.githubusercontent.com/49491949/109721147-f9130f80-7b78-11eb-9f51-225e1c69993d.png)
to this:
![image](https://user-images.githubusercontent.com/49491949/109721618-9706da00-7b79-11eb-8c35-eaa30ff06bd5.png)
the pop up error disappears

Reply:
Thanks, @JillieBeanSim I added the checking and I hope it will fix the issue. I did have the jobs in favorites so I did not encounter it previously.